2024/2024-ssl-passingAndShooting-rules.adoc

```diff
@@ -0,0 +1,90 @@
+members, officials, etc. are for simplification and apply to both males and
+females.
+
+== Goals of the Technical Challenge
+
+In this challenge, the central objective is to test the ability of a set of attackers to score a goal by making at least one pass to an allied robot and finishing the play with a shot at the opponent's goal.
+
+Within a limited timeframe, the team needs to make as many successful passes as possible, culminating in a shot at the opponent's goal to complete the play. This challenge not only assesses the participants' technical skills but also their ability to work as a team and make quick, practical decisions during the match.
+
+By focusing on the precise execution of passes, collaboration between robots, and finishing the play with a shot at the opponent's goal, this challenge encourages innovation and continuous development in the SSL - Regular League.
+
+=== Participation Requirements
+
+All teams of the Regular League are eligible and *are encouraged* to participate in this challenge.
+
+== Procedure
+There will be five different situations in random order that teams have to handle. The situations include the positions of the opposing robots and the ball. Put up to robots on the field and score as many goals as possible within 300 seconds.
+
+* To start, all robots must be placed on the middle line.
+* Opponent robots will be placed on the field as obstacles.
+* A *STOP* command will be issued for exactly 5s to allow the team to prepare their robots.
+* Immediately afterward, a *FORCE START* command will be issued and the timer will start.
+* A goal can be scored only when the kicking robot is in the opponent’s half.
+* The team has *20s* to score a goal. Afterward, the trial is over.
+* As soon as a goal is scored or the ball leaves the field, it will be placed again in one of the defined situations.
+* Every team has to perform the three trials of each scenario or can withdraw a trial, counting 20s.
+* After 5min a *HALT* command will be issued and the challenge is over.
+
+=== Pass Definitions 
+Teams are free in how they perform the passes. Definitions of a pass in general and additional constraints for valid pass that are considered in the scoring are given in this section.
+
+A pass is defined as follows:
+
+* A pass has a source position, a target position, and a shooter.
+* The source position is where a robot kicks the ball.
+* The target position is where the ball stops or changes direction by more than *30 degrees* in the field plane (x, y).
+* The shooter is the robot located nearest to the source position when the pass starts.
+* The pass distance is the Euclidean distance between the source and target position.
+* The pass direction is the orientation of the `target - source` vector.
+* The pass is considered *chipped*, if the ball reached a maximum height of at least *15cm*.
+
+NOTE: Only passes and shots made using the kicker will be considered.
```

Comment:
For the regular league, I really think it's absurd to put this in the rules.

Reply:
That is a fair point, I only say this because there isn't a formal definition of what is a kicking device in the ssl-rules (for me it is ok to not exist, I'm just saying to be clear for us members of the committee).

Reply:
How would we define a kick mechanism? I think we shouldn't worry about that.

Reply:
Ok